### PR TITLE
New Punjabi Language Cheat Sheet

### PR DESCRIPTION
**Pull request by sinscary**
Punjabi Language Cheatsheet

IA Page: [https://duck.co/ia/view/punjabi_cheat_sheet](https://duck.co/ia/view/punjabi_cheat_sheet)

### CODE REVIEW

**Comment by daxtheduck** [comment]
@sinscary Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/punjabi.json [hunk 1]
```diff
@@ -0,0 +1,326 @@
+{
+    "id": "punjabi_cheat_sheet",
+    "name": "Punjabi Cheat Sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/punjabi.json]: Please remove "Cheat Sheet"

File: share/goodie/cheat_sheets/json/punjabi.json [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+    "description": "This is cheatsheet for basic but mostly used in daily conversation phrases in punjabi",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/punjabi.json]: Let's go with something like "List of conversational Punjabi phrases"

File: share/goodie/cheat_sheets/json/punjabi.json [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+
+    "metadata": {
+        "sourceName": "Omniglot",
+        "sourceUrl" : "http://www.omniglot.com/language/phrases/punjabi.php"
+    },
+
+    "aliases": [
+        "punjabi language", "language used in punjab", "punjab", "gurumukhi", "punjabi dictionary", "english to punjabi", "learn punjabi", "general punjabi"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/punjabi.json]: You can remove all these -- we have builtin aliases for most of these phrases now :+1:

**Comment by moollaza** [comment]
@sinscary thanks for contributing! Are you a native Punjabi Speaker?

@duckduckgo/community-leaders any of you speak Punjabi?

**Comment by codenirvana** [comment]
@moollaza I don't think anyone from us know Punjabi, Need to find some contributor who can help in reviewing this.

**Comment by sinscary** [comment]
@moollaza Thanks and yes i am a native punjabi speaker. I am from Punjab.

**Comment by moollaza** [comment]
@codenirvana thanks for letting me know!

@sinscary Thanks! This LGTM :+1: Seeing as you're a native speaker I'll merge this

**Comment by sinscary** [comment]
@moollaza Thanks and will i be getting any goodies, i mean where is my DDG t-shirt and stickers :) :p :+1:  .

**Comment by sinscary** [comment]
@moollaza Thanks :+1: , anyways where is my t-shirt and stickers. :p :)

**Comment by moollaza** [comment]
@sinscary thanks again for creating the Punjabi Cheat Sheet! You’ve done a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in Punjabi.

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to Punjabi

OR

- Test live Punjabi Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Comment by sinscary** [comment]
@moollaza  sure i will. :+1:

**Comment by moollaza** [comment]
> @moollaza sure i will.

@sinscary awesome, thanks! Are there any punjabi related queries you can think of that don't have a good Instant Answer? Or would you like to share this cheat sheet online with others to gather feedback?

**Comment by sinscary** [comment]
@moollaza  sure i would like to share this online and yes there are many queries like punjabi food etc. remaining i will let you know also i will love to make a cheatsheet.

**Pull request merged**